pkg/experiment/metadata.go

```diff
@@ -202,6 +203,16 @@ func (m *Metadata) RecordMap(metadata MetadataMap) error {
 	return m.storeMap(metadata, metadataKindEmpty)
 }
 
+// RecordTags saves information about tags that experiments is going to generate,
+// that one can automatically build a visualization of results.
+func (m *Metadata) RecordTags(tags []string) error {
```

Comment:
I would even add `RecordVisualisationDimensions` to give explicitly connect this function with pivot table visualisation.